doc/source/conf.py

```diff
@@ -44,11 +45,36 @@
     'sphinx_gallery.gen_gallery',
 ]
 
+# -- sphinx_gallery configuration
+from sphinx_gallery.gen_gallery import ExplicitOrder
+
+sg_gallery_dir = 'auto_examples'
+sg_examples_dir = '../../examples'
+
+def subsection_title_order(src_dir: str):
+    """
+    src_dir : str
+        string name of source path
+    """
+
+    sub_folders = [p.name for p in Path(src_dir).iterdir() if p.is_dir()]
+    final_order = []
+    for sub_folder in sub_folders:
+        subdir_name = f"{src_dir}/{sub_folder}"
+        if sub_folder == "release_highlights":
+            final_order.insert(0, subdir_name)
+        else:
+            final_order.append(subdir_name)
+    return ExplicitOrder(final_order)
```

Comment:
I'm not *super* keen on this as it adds some not super necessary complexity though I see the benefit. 
At least we should make that function recursive in case we add more than one level of nesting.

Reply:
I'll remove it for now until useful